alluxio-ui/common/src/constants/data/footerNavigation.tsx

```diff
@@ -9,7 +9,7 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-import {INavigationData} from '@alluxio/common-ui/src/constants';
+import {INavigationData} from '..';
```

Comment:
Are relative paths preferred?

Reply:
yeah, there's no need to import from the package if it's local to the package

Reply:
This also happened because I moved the file, it was an external resource before (master-ui and worker-ui were including it from the common-ui package)

Reply:
I think that I might need to undo this... I think that the workers link in the workers footer should probably link to the master's /workers page.